components/PrimarySourceSetsComponents/Source/components/ContentAndMetadata/index.js

```diff
@@ -52,6 +59,32 @@ const getViewerComponent = (fileFormat, type, pathToFile) => {
 const getDomainFromThumbnail = thumbnailUrl =>
   /^(?:https?:)?(\/\/[\w.]+\/)/.exec(thumbnailUrl)[1];
 
+// TODO: Make this a utilFunction and use in all Click Through events.
+const trackClickThrough = (e, source, target = "_blank") => {
+  const href = getSourceLink(source);
+
+  const gaEvent = {
+    type: "Click Through",
+    itemId: getItemId(source),
+    title: joinIfArray(getTitle(source)),
+    partner: joinIfArray(getPartner(source)),
+    contributor: joinIfArray(getContributor(source))
+  };
+
+  // e is a React synthetic event
+  e.preventDefault();
+
+  try {
+    // Try tracking a Google Analytics event.
+    trackGaEvent(gaEvent);
+  } catch (error) {
+    // TODO: What is the best way to log an error?
```

Comment:
There's the console, but we should probably be looking to use something like HoneyBadger or AWS X-Ray.